### ISSUE TITLE
Implement a Camping Simulator page

### ISSUE THREAD

**Issue by RaphaelDDL** [EpicSevenDB/ui#12]
We already have the data for the camping, so it's just a matter of coding the page.

**Comment by willhausman** [comment]
Did you have a design in mind for this page?  In my mind, it sounds neat to display 4 empty camping slots, and as each slot is filled with a hero, continually re-calculate recommended values.  I'm thinking a custom "dropdown" that looks similar to the game's sliding list of hero cards on the right.  On a phone, let the list take up the whole page until selection dismissal. 

![image](https://user-images.githubusercontent.com/2905110/54958494-cc353700-4f23-11e9-9622-1f958c905a47.png)

Edit: and a quicksearch filter at the top of the list to find the hero you're after.  Of course, the page URL can supply hero ids to auto populate the slots.

**Comment by RaphaelDDL** [comment]
I wanted to do it easier actually, with the vue-multiselect. Using the buff/debuff as example (but without multi option, only single option per select):

![image](https://user-images.githubusercontent.com/1087109/55114588-1a337180-50c1-11e9-999d-1339969786dd.png)

This is a custom template slot option. So in heroes' case, it would be 4 selects having the option be the circle image with the name. 
![image](https://user-images.githubusercontent.com/1087109/55114710-6d0d2900-50c1-11e9-93ea-ba98244e91d8.png)

While having 4 selects is more clicks than one with multiple options, It's easier to understand.

**Comment by willhausman** [comment]
Seems like the same thing to me, just a simplified custom template slot option to be a circle image instead of the blue with a small image.  My two images show where you've only selected one, and then where you've clicked on the second one and are flipping through the select options to choose the second one.  It sounds like we're on the same page for how to select them, just different pages for what the templates look like.

**Comment by willhausman** [comment]
I had put some time into this, but got busy and didn't get to complete it.  Since then, I've spotted links here and there on reddit that point to a camping tool and optimizer.  Perhaps this issue should be scrapped and a link to one or more of those tools is the better approach.

**Comment by RaphaelDDL** [comment]
I agree. There's a korean one which is great. Even YufineBot on discord has too.

I'll add as low priority because I want to add this to the website at some point, the calculation is fixed anyway so would be a nice to have for later.
